catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/converter/PostgreSqlTypeConverter.java

```diff
@@ -7,87 +7,83 @@
 import com.datastrato.gravitino.catalog.jdbc.converter.JdbcTypeConverter;
 import com.datastrato.gravitino.rel.types.Type;
 import com.datastrato.gravitino.rel.types.Types;
-import java.util.List;
-import net.sf.jsqlparser.statement.create.table.ColDataType;
 
-public class PostgreSqlTypeConverter extends JdbcTypeConverter<ColDataType, String> {
+public class PostgreSqlTypeConverter extends JdbcTypeConverter<String> {
   @Override
-  public Type toGravitinoType(ColDataType type) {
-    List<String> arguments = type.getArgumentsStringList();
+  public Type toGravitinoType(JdbcTypeBean typeBean) {
     // TODO #947 Complex types are not considered for support in this issue, which will bring more
     // testing needs
-    switch (type.getDataType().toLowerCase()) {
-      case "boolean":
+    switch (typeBean.getTypeName().toLowerCase()) {
+      case "bool":
         return Types.BooleanType.get();
-      case "smallint":
+      case "int2":
```

Comment:
Can you use a static parameter for the const string?

Reply:
+1